lib/internal/per_context/domexception.js

```diff
@@ -49,14 +49,23 @@ const disusedNamesSet = new SafeSet()
   .add('ValidationError');
 
 class DOMException {
-  constructor(message = '', name = 'Error') {
+  constructor(message = '', name = 'Error', cause = undefined) {
     ErrorCaptureStackTrace(this);
     internalsMap.set(this, {
       message: `${message}`,
-      name: `${name}`
+      name: `${name}`,
+      cause
     });
```

Comment:
Wouldn't that be more correct? (at least that's the `'cause' in new Error === false`, and `'cause' in new Error('', { cause: undefined})) === true`)
```suggestion
    });
    if (arguments.length > 2) {
      ObjectDefineProperty(this, 'cause', {
        __proto__: null,
        configurable: true,
        enumerable: true,
        value: cause,
        writable: true,
      });
    }
```